shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -754,6 +770,8 @@ - (void)hideTextInput {
 
 - (void)setTextInputClient:(int)client withConfiguration:(NSDictionary*)configuration {
   NSDictionary* inputType = configuration[@"inputType"];
+  if(@available iOS (11.0,*))
+      NSDictionary* contentType = configuration[@"contentType"];
```

Comment:
It looks like this is an unused variable.

Reply:
This variable is referred by https://github.com/flutter/flutter/issues/33040

Reply:
> It looks like this is an unused variable.

Maybe I will check CI After Exam Month